createdisk-library.sh

```diff
@@ -7,6 +7,7 @@ function get_dest_dir {
     if [ "${OPENSHIFT_VERSION-}" != "" ]; then
         DEST_DIR=$OPENSHIFT_VERSION
     else
+        DEST_DIR=${PULL_NUMBER}
         set +e
         DEST_DIR=$(git describe --exact-match --tags HEAD)
```

Comment:
This overwrites the DEST_DIR you just set?

Reply:
@cfergeau Ah, anyway `DEST_DIR=$(git describe --exact-match --tags HEAD)` is not used anymore so better to remove it all together.